tests/test_logging.py

```diff
@@ -0,0 +1,111 @@
+from datetime import datetime
+import os
+
+
+def test_log():
+    """Test basic log operation outside model operation"""
+    from tm2py.config import LoggingConfig
+    import tempfile
+
+    with tempfile.TemporaryDirectory() as temp_dir:
+        log_config = {
```

Comment:
this doesn't have to be in tempfile context

Reply:
Moved